src/components/interaction-designer/block-editors/GroupMembershipEditor.vue

```diff
@@ -0,0 +1,164 @@
+<template>
```

Comment:
I adopted the same approach from https://github.com/FLOIP/flow-builder/blob/master/src/components/interaction-designer/block-editors/ContactPropertyEditor.vue